diff --git a/node_modules/chrome-aws-lambda/build/index.js b/node_modules/chrome-aws-lambda/build/index.js
index f78e183..cc4ada6 100644
--- a/node_modules/chrome-aws-lambda/build/index.js
+++ b/node_modules/chrome-aws-lambda/build/index.js
@@ -7,7 +7,7 @@ const fs_1 = require("fs");
 const lambdafs_1 = __importDefault(require("lambdafs"));
 const path_1 = require("path");
 const url_1 = require("url");
-if (/^AWS_Lambda_nodejs(?:10|12|14)[.]x$/.test(process.env.AWS_EXECUTION_ENV) === true) {
+if (/^AWS_Lambda_nodejs(?:10|12|14|16)[.]x$/.test(process.env.AWS_EXECUTION_ENV) === true) {
     if (process.env.FONTCONFIG_PATH === undefined) {
         process.env.FONTCONFIG_PATH = '/tmp/aws';
     }
@@ -145,7 +145,7 @@ class Chromium {
             lambdafs_1.default.inflate(`${input}/chromium.br`),
             lambdafs_1.default.inflate(`${input}/swiftshader.tar.br`),
         ];
-        if (/^AWS_Lambda_nodejs(?:10|12|14)[.]x$/.test(process.env.AWS_EXECUTION_ENV) === true) {
+        if (/^AWS_Lambda_nodejs(?:10|12|14|16)[.]x$/.test(process.env.AWS_EXECUTION_ENV) === true) {
             promises.push(lambdafs_1.default.inflate(`${input}/aws.tar.br`));
         }
         return Promise.all(promises).then((result) => result.shift());
